.gitignore

```diff
@@ -7,7 +7,7 @@
 /coverage
 
 # production
-/build
```

Comment:
i don't think we should be committing our builds, i think that when we deploy a new version we should build and deploy there.

If we include built files we're going to get useless merge conflicts on nearly every pr

Reply:
can you add a PR for instructions for deploying?

Reply:
We can either use GitHub releases or go back to the npm releases, but committing built files is a no-go.

I'll work on the doc for publishing tomorrow.